src/plugins/intel_gna/src/gna_plugin.cpp

```diff
@@ -376,27 +384,36 @@ void GNAPlugin::Init() {
     gnaFlags = std::make_shared<GNAFlags>(GNAFlags());
     inputs_ptr_ = std::make_shared<GnaInputs>(GnaInputs());
     outputs_ = GnaOutputs();
-
-    graphCompiler.setDNNPtr(dnn);
-    graphCompiler.setInputsPtr(inputs_ptr_);
-
     requestWorkerPool_ = std::make_shared<request::WorkerPoolImpl>();
 }
 
 void GNAPlugin::InitGNADevice() {
     OV_ITT_SCOPED_TASK(itt::domains::GNA_LT, "InitGNADevice");
-    if (gnaFlags->sw_fp32) {
-        gnamem.reset(new gna_memory_float(memory::GNAFloatAllocator{}));
-    } else {
+
+    if (!gnaFlags->sw_fp32) {
         gnadevice = std::make_shared<GNADeviceHelper>(config.target,
                                                       gnaFlags->performance_counting,
                                                       !config.embedded_export_path.empty());
+    }
+}
+
+void GNAPlugin::InitGNAMemory() {
+    OV_ITT_SCOPED_TASK(itt::domains::GNA_LT, "InitGNAMemory");
 
+    if (gnaFlags->sw_fp32) {
+        gnamem.reset(new gna_memory_float(memory::GNAFloatAllocator{}));
+    } else {
         gnamem = std::make_shared<gna_memory_device>(memory::GNAAllocator(gnadevice),
-                                                     gnadevice->getMemAlignment(),
-                                                     limitations::kMemoryPageSize);
+                                                     Limitations::get_instance()->get_memory_alignment(),
+                                                     Limitations::kMemoryPageSize);
     }
-    graphCompiler.setGNAMemoryPtr(gnamem);
+}
+
+void GNAPlugin::InitGraphCompiler() {
+    OV_ITT_SCOPED_TASK(itt::domains::GNA_LT, "InitGraphCompiler");
+
+    m_graph_compiler = std::make_shared<GNAGraphCompiler>(
+        GNAGraphCompiler(config, dnn, inputs_ptr_, Limitations::get_instance()->get_cnn_validator(), gnamem));
```

Comment:
is formatting applied properly?

Reply:
I am using configuration from our coding style guide, also CI does not report any issues